### PR TITLE
sepolicy: Allow mflinger to access gpu device

### PR DESCRIPTION
**Pull request by utzcoz**
The new rules are got from sargo device.

### CODE REVIEW

File: sepolicy/mflinger.te [hunk 1]
```diff
@@ -18,3 +18,6 @@ allow mflinger surfaceflinger_service:service_manager find;
 
 # allow mflinger to read the ion buffer used by mclient
 allow mflinger ion_device:chr_file r_file_perms;
+allow mflinger gpu_device:chr_file { read write };
+allow mflinger vendor_default_prop:file { getattr open read };
+allow mflinger vendor_display_prop:file { getattr open read };
```
Comment (utzcoz) [inline sepolicy/mflinger.te]: vendro_default_prop and vendor_display_prop are not defined by sailfish. I will check whether they are needed, or I will move those properties to sargo/bonito device's sepolicy directory.

Comment (pdsouza) [reply sepolicy/mflinger.te]: OK sounds good, thanks for checking.

Comment (utzcoz) [reply sepolicy/mflinger.te]: They are not needed. I removed it from new commit.

**Comment by utzcoz** [comment]
I will test it with marlin running and pme building. @pdsouza could you help to test building for Nexus 5? Thanks.

**Comment by utzcoz** [comment]
> I will test it with marlin running and pme building. @pdsouza could you help to test building for Nexus 5? Thanks.

It works for marlin device. But pme building failed with command `mka dist` with unresolved dependencies, not related to sepolicy rules. I will copy source code to a single directory to test clean building.

**Comment by pdsouza** [comment]
> > I will test it with marlin running and pme building. @pdsouza could you help to test building for Nexus 5? Thanks.
> 
> It works for marlin device. But pme building failed with command `mka dist` with unresolved dependencies, not related to sepolicy rules. I will copy source code to a single directory to test clean building.

Any chance this happened because of the other PR I merged in dealing with VNDK?

I'll test on my N5 as well this week.

**Comment by utzcoz** [comment]
> Any chance this happened because of the other PR I merged in dealing with VNDK?

From the error info, it is not related to it, because it doesn't show information about `lxc`. It is also failed with clean build. I will revert it locally to test it again.

**Comment by utzcoz** [comment]
> > Any chance this happened because of the other PR I merged in dealing with VNDK?
> 
> From the error info, it is not related to it, because it doesn't show information about `lxc`. It is also failed with clean build. I will revert it locally to test it again.

@pdsouza It is also failed with reverted lxc commit. Hi @pintaf could you build pme succeed locally?

**Comment by utzcoz** [comment]
My local error info is here:

```
external/ant-wireless/ant_native/Android.mk: error: "antradio_app (EXECUTABLES android-arm64) missing libantradio (SHARED_LIBRARIES android-arm64)" 
```

**Comment by pintaf** [comment]
Hi @utzcoz It's been a long time I did not attempt to build maru.
My pme is my daily phone but I'm in the process of changing, I'm just porting /e/ os to my future phone. Once done I'll resume working on maru and will answer your questions and participate in dev again.

**Comment by pdsouza** [comment]
@utzcoz I am running a build for `pme` right now. I'll let you know soon. I need to verify it works to get automated builds for `pme` working anyways 😁

**Comment by pdsouza** [comment]
> My local error info is here:
> 
> ```
> external/ant-wireless/ant_native/Android.mk: error: "antradio_app (EXECUTABLES android-arm64) missing libantradio (SHARED_LIBRARIES android-arm64)" 
> ```

I have the same error building `pme`. @utzcoz this is not due to your changes since I have not called `repo sync` yet.

**Comment by utzcoz** [comment]
@pdsouza Looks like it not related to recent PRs, what about merging this PR and opening a new issue to track pme's build problem?

**Comment by pdsouza** [comment]
> @pdsouza Looks like it not related to recent PRs, what about merging this PR and opening a new issue to track pme's build problem?

Sounds good. Tracking the pme issue in https://github.com/maruos/android_device_htc_pme/issues/1.

**Comment by utzcoz** [comment]
@pdsouza The next thing is to send PR for manifest to track sargo device and kernel.

**Comment by pdsouza** [comment]
I'm ready to merge when it's ready!

**Pull request merged**